test/tests/functional/pbs_node_rampdown.py

```diff
@@ -151,6 +150,32 @@ def pbs_nodefile_match_exec_host(self, jid, exec_host,
             return False
         return True
 
+    def check_test_img_size(self):
```

Comment:
rename the function to the effect that it refers stageout file instead of test image